### ISSUE TITLE
7 box display - Physical bject

### ISSUE THREAD

**Issue by HymaKanchiAU** [aehrc/snomio#406]
displaying a concept which is a device type the labels do not match the semantic tags of the FSN
Search for 45131000036105 SCT ID
<img width="757" alt="image" src="https://github.com/aehrc/snomio/assets/126848487/4eef05e2-971a-4b59-baae-372365aaccc9">

**Comment by ChrisKellaleaMaynard** [comment]
Updated title and description to add A/C and clarifications for this issue.

**Comment by dionmcm** [comment]
This is tricky...the "boxes" really represent concepts from the bottom most "containerised" product concept's concept graph which are in the AMT reference sets. These reference sets are MP, MPUU, TP, TPUU, MPP, TPP, and CTPP.

While we don't want to propagate these refset boxes, that's essentially what this diagram is.

So for a drug product, the names correlate pretty well between the refsets and the semantic tags. For example Medicinal product unit of use and Clinical drug.

For a device, this doesn't really work. But these device concepts _are_ in the reference sets as shown. So despite being "physical object"s these concepts are in the MP, MPUU...etc reference sets, and shown in those corresponding boxes on this screen.

I think we need to have a chat with @MattCordell, Sarah, Linda, and @kylynnloi to decide what is "right" in this view. Maybe the boxes should be labelled in terms of the refsets, which is what they represent? Would that help? It has the unfortunate effect of prolonging the notion behind these refsets...

Bottom line this needs more discussion.

**Comment by ChrisKellaleaMaynard** [comment]
@dionmcm   The notion behind this issue was for the API to determine the associated "type" of concept (MP, TPUU, etc) and pass it back to the UI.  If this occurs then the UI can simply post the semantic tag as the heading for the box instead of having it hardcoded.  This then allows other taxonomical approaches (e.g. devices, or for other NRCs) to be used and no redesigning of the screen's headings would be required.  The trick is, as you've described above, if the tags are generated based on the reference sets they're retrieved from, then another approach may be required.

**Comment by MattCordell** [comment]
Although only a lebeling thing, it is messy...
The 7 box thing, is very AMT centric, and is what it's trying to show here.. so maybe the refset names are correct.
However, would this be applicable to other NRCs? And as Dion said, do we want to perpetuate the refsets? If so, we're never going to be able to get rid of the TPUU/TPP/CTPP duplicates...

The most abstract thing is something like 
|Product|Brand Name|
|Generic Product|Branded Product|
|Packaged Product| Branded Package|
||Branded Container|

Though this introduces another set of names to confuse everyone...
Perhaps, we just scrap the labels? Are they necessary for the UI? It's always that "7 box pattern". Can the user just work it out?
As a minimum I feel like we should focus primarily on our requirements (AMT) for now, and not worry about other users.

**Comment by kylynnloi** [comment]
updated ticket to note that we will use "generic labels" as per MC's suggestion.

**Comment by ChrisKellaleaMaynard** [comment]
Looks good, except where it says "Brand Pack" on the screen instead of "Branded Pack".

![Image](https://github.com/aehrc/snomio/assets/140361342/55bbb44d-a26c-4223-b9bc-079a53b8d4c6)

**Comment by senjo-aehrc** [comment]
@ChrisKellaleaMaynard, its fixed and available in dev snomio

**Comment by ChrisKellaleaMaynard** [comment]
Retested and passed.  Moving to Testing.
